test/integration/targets/setup_mongodb/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+mongodb_version: "4.0"
+
+apt_xenial:
+  keyserver: "keyserver.ubuntu.com"
+  keyserver_id: "2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5"
+  repo: "deb [ arch=amd64 ] http://repo.mongodb.org/apt/ubuntu {{ansible_distribution_release}}/mongodb-org/{{mongodb_version}} multiverse"
+
+apt_bionic:
+  keyserver: "keyserver.ubuntu.com"
+  keyserver_id: "9DA31620334BD75D9DCB49F368818C72E52529D4"
+  repo: "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu {{ansible_distribution_release}}/mongodb-org/{{mongodb_version}} multiverse"
+
+mongodb_packages:
+  mongod: mongodb-org-server
+  mongos: mongodb-org-mongos
+  mongo: mongodb-org-shell
+
+additional_packages:
+  - python-pip
+  - python-pymongo
```

Comment:
Install `pymongo` using pip instead of OS packages.

Reply:
Corrected.